packages/appeals-service-api/src/repositories/appeals-repository.js

```diff
@@ -71,6 +71,14 @@ class AppealsRepository extends MongoRepository {
 				{ returnDocument: 'after' }
 			);
 	}
+
+	/**
+	 * @param {string} appealId
+	 * @return {Promise<any>}
+	 */
+	async delete(appealId) {
+		return await mongodb.get().collection(this.collectionName).deleteOne({ _id: appealId });
```

Comment:
guessing the findOneAndDelete or whatever it was before was the issue?

Reply:
I was doing appeal.id here but only had the id passed through
looked into the docs a little harder before realising that and I believe findOneAndDelete returns the deleted document which we didn't need so switched to the simpler function